.github/workflows/build.yml

```diff
@@ -14,12 +14,12 @@ jobs:
     steps:
 
       - name: Checkout code
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
```

Comment:
Just updated these as older actions will stop working eventually (some nodejs thing that GitHub deprecated).